problem2.py

```diff
@@ -1 +1,8 @@
+first_side = float(input("Enter first side: "))
+second_side = float(input("Enter second side: "))
+third_side = float(input("Enter third side: "))
 
+if first_side + second_side > third_side or second_side + third_side > first_side or first_side + third_side > second_side:
```

Comment:
` triangle when the sum of any two sides is greater than the third side` --> should be using `and` instead of `or`